rustls/src/manual/mod.rs

```diff
@@ -29,6 +29,7 @@ pub mod _04_features;
 #[path = "defaults.rs"]
 pub mod _05_defaults;
 
-/// This section provides guidance on using rustls with FIPS-approved cryptography.
-#[path = "fips.rs"]
-pub mod _06_fips;
+// NOW REMOVED FROM THIS FORK:
+// /// This section provides guidance on using rustls with FIPS-approved cryptography.
+// #[path = "fips.rs"]
+// pub mod _06_fips;
```

Comment:
This feels a little inconsistent with some of the other updates and keeps `rustls/src/manual/fips.rs` in place as an orphaned module. I think it would be best to add a comment to the beginning of `rustls/src/manual/fips.rs` that this documentation module is orphaned in this fork.